packages/esm-service-queues-app/src/add-provider-queue-room/add-provider-queue-room.component.tsx

```diff
@@ -61,7 +61,7 @@ const AddProviderQueueRoom: React.FC<AddProviderQueueRoomProps> = ({ providerUui
 
   const { mutate } = useProvidersQueueRoom(providerUuid);
   const { queues } = useQueues(currentLocationUuid);
-  const { rooms } = useQueueRooms(currentLocationUuid, currentServiceUuid);
+  const { rooms } = useQueueRooms(currentLocationUuid, currentLocationUuid);
```

Comment:
we are passing the same parameter. Is there a possibility to filter by service as well?